### ISSUE TITLE
When Inheriting SESlidedonot add a method called setUp. [Not an issue]

### ISSUE THREAD

**Issue by sai-prasanna** [spaceelephant/SESlideTableViewCell#12]
I was inheriting SESlideTableViewCell, strangest thing happened, I had a method called setUp defined and called from init , but compiler did not detect it as an override (may be because it is internal to the .m file?) . While running the setUp defined in SESlideTableViewCell was not called, rather the one which unofficially overrides it gets called. Making the method private , will fix the issue. 

Is this swift bug, or how it is designed to do stuff?

I thought it was because, I had started to use autolayout programatically (using another wonderful lib https://github.com/s4cha/Stevia) and SESlide did not support autolayout for frames, but I was wrong. SESSlide is as wonderful as ever, it supports autolayout.

TLDR; Do not have a non private method named setUp defined in your class inheriting SESlideTableViewCell
